torch/utils/cpp_backtrace.py

```diff
@@ -9,3 +12,19 @@ def get_cpp_backtrace(frames_to_skip=0, maximum_number_of_frames=64) -> str:
         maximum_number_of_frames (int): the maximum number of frames to return
     """
     return _get_cpp_backtrace(frames_to_skip, maximum_number_of_frames)
+
+def get_python_cpp_trace():
```

Comment:
cc @ezyang this feels like it overlaps with
```
from torch.utils._traceback import CapturedTraceback
print(''.join(CapturedTraceback.extract(cpp=True).format()))
```
that you advertised before. Should it re-use the logic from there directly?

Reply:
Yeah, it's the same thing. The value add is that @weifengpy is doing frame filtering. This is a good idea and we should have this as an option in CapturedTraceback too.

Reply:
thanks. good to know it's already implemented in `torch._traceback`. will close this PR for now. i will maybe add an option in CapturedTraceback for filtering